bot/cogs/random_cog/random_cog.py

```diff
@@ -92,5 +93,44 @@ async def ball(self, ctx, *, question):
         embed = discord.Embed(title='🎱', description= f'{random.choice(responses)}',color = Colors.ClemsonOrange)
         await ctx.send(embed=embed)
 
+    @commands.command(aliases=['slotmachine','🎰'])
+    @commands.cooldown(1, SLOTS_COMMAND_COOLDOWN, commands.BucketType.guild)
```

Comment:
Tbh i dont see the need for a command cooldown

Reply:
I just added it because it looks really odd when its running if multiple people call it at once. I can remove it though.

Reply:
This can be fixed with the edit change i mentioned below, instead of a new embed we edit each persons original one. its a much more natural look